packages/api/src/external/commonwell/__tests__/cw-patient-data.ts

```diff
@@ -18,7 +18,7 @@ export function makeCwDataLink(): CwLink {
   const address = makeAddressStrict();
   return {
     _links: {
-      self: {
+      [faker.helpers.arrayElement(["upgrade", "downgrade"])]: {
```

Comment:
Shouldn't we keep `self` in addition to the other ones?

Reply:
self actually never appears in the first 2281 CW links. But, I also fixed this to be reflective of assuranceLevel (1 -> upgrade and downgrade, 2 -> just downgrade, 3 -> nothing).